### ISSUE TITLE
Make RS/FS configurable for CSV

### ISSUE THREAD

**Issue by johnkerl** [johnkerl/miller#50]
It will still be RFC4180 by default, but:
* Option to set `RS`, e.g. to CR or LF rather than standard CRLF;
* Option to set `FS`, e.g. for TSV
* As a necessary side effect, `RS`/`FS`/`PS` for all formats will be able to be multi-character, e.g. you'll be able to use CRLF for DKVP format which will resolve https://github.com/johnkerl/miller/issues/19.

**Comment by johnkerl** [comment]
This is functionally complete for CSV as of https://github.com/johnkerl/miller/commit/1ff438ffb23a88cdf0342210e7c63031e521fdfc.

Remaining to do:
* Expanded unit-test coverage, HTML docs, on-line help. Then, I'll close this issue.
* Unlike as I'd originally planned, I'll add multi-character-separator support for other file formats as a separate task: see https://github.com/johnkerl/miller/issues/19.

**Comment by johnkerl** [comment]
Closed by commits ending in https://github.com/johnkerl/miller/commit/6879eecf9ef10fb6eeb1377dab142b740b18f6ca. Online help updated.

TL;DR: `mlr --csv --rs lf ...` to handle native Linux files. (As always, default CSV is RFC4180-compliant -- these RS/FS options allows you to specify some non-compliance when desired.)

**Comment by johnkerl** [comment]
This feature is worthy of a release, for sure, but I'll wait to include https://github.com/johnkerl/miller/issues/19 as well.
